typescript/sdk/src/providers/ChainConnection.ts

```diff
@@ -38,7 +38,7 @@ export class ChainConnection {
   }
 
   getApiUrl(): string {
-    return this.blockExplorerApiUrl;
+    return this.blockExplorerApiUrl + '/api';
```

Comment:
Do you need to change https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/sdk/src/deploy/middleware/LiquidityLayerApp.ts#L77 and https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/main/typescript/sdk/src/deploy/middleware/LiquidityLayerApp.ts#L91 too?